server/views/pages/inPrisonCoursesAndQualifications/index.njk

```diff
@@ -51,9 +51,7 @@
             {{ sortableTable({
               id: "completed-courses",
               courses: completedCourses,
-              locationColumn: true,
-              completedOnColumn: true,
-              gradeColumn: true
+              courseStatus: "COMPLETED"
```

Comment:
I've refactored the way the `sortableTable` macro works here, in that now it takes the "type" of course data it is displaying, rather than controlling them with booleans
It makes the readability of the `sortableTable` itself easier (see below)